diff -urN sonarqube-5.3/bin/linux-x86-32/sonar.sh sonarqube-5.3-mod/bin/linux-x86-32/sonar.sh
--- sonarqube-5.3/bin/linux-x86-32/sonar.sh	2016-02-23 23:05:37.000000000 +0800
+++ sonarqube-5.3-mod/bin/linux-x86-32/sonar.sh	2016-02-23 23:14:04.000000000 +0800
@@ -22,15 +22,15 @@
 APP_LONG_NAME="${DEF_APP_LONG_NAME}"
 
 # Wrapper
-WRAPPER_CMD="./wrapper"
-WRAPPER_CONF="../../conf/wrapper.conf"
+WRAPPER_CMD="/usr/share/sonar/bin/wrapper"
+WRAPPER_CONF="/etc/sonar/wrapper.conf"
 
 # Priority at which to run the wrapper.  See "man nice" for valid priorities.
 #  nice is only used if a priority is specified.
 PRIORITY=
 
 # Location of the pid file.
-PIDDIR="."
+PIDDIR="/var/run/sonar"
 
 # If uncommented, causes the Wrapper to be shutdown using an anchor file.
 #  When launched with the 'start' command, it will also ignore all INT and
@@ -45,7 +45,7 @@
 # NOTE - This will set the user which is used to run the Wrapper as well as
 #  the JVM and is not useful in situations where a privileged resource or
 #  port needs to be allocated prior to the user being changed.
-#RUN_AS_USER=
+RUN_AS_USER=sonar
 
 # The following two lines are used by the chkconfig command. Change as is
 #  appropriate for your application.  They should remain commented.
@@ -561,4 +561,4 @@
         ;;
 esac
 
-exit 0
\ No newline at end of file
+exit 0
diff -urN sonarqube-5.3/bin/linux-x86-64/sonar.sh sonarqube-5.3-mod/bin/linux-x86-64/sonar.sh
--- sonarqube-5.3/bin/linux-x86-64/sonar.sh	2016-02-23 23:05:37.000000000 +0800
+++ sonarqube-5.3-mod/bin/linux-x86-64/sonar.sh	2016-02-23 23:15:39.000000000 +0800
@@ -22,15 +22,15 @@
 APP_LONG_NAME="${DEF_APP_LONG_NAME}"
 
 # Wrapper
-WRAPPER_CMD="./wrapper"
-WRAPPER_CONF="../../conf/wrapper.conf"
+WRAPPER_CMD="/usr/share/sonar/bin/wrapper"
+WRAPPER_CONF="/etc/sonar/wrapper.conf"
 
 # Priority at which to run the wrapper.  See "man nice" for valid priorities.
 #  nice is only used if a priority is specified.
 PRIORITY=
 
 # Location of the pid file.
-PIDDIR="."
+PIDDIR="/var/run/sonar"
 
 # If uncommented, causes the Wrapper to be shutdown using an anchor file.
 #  When launched with the 'start' command, it will also ignore all INT and
@@ -45,7 +45,7 @@
 # NOTE - This will set the user which is used to run the Wrapper as well as
 #  the JVM and is not useful in situations where a privileged resource or
 #  port needs to be allocated prior to the user being changed.
-#RUN_AS_USER=
+RUN_AS_USER=sonar
 
 # The following two lines are used by the chkconfig command. Change as is
 #  appropriate for your application.  They should remain commented.
@@ -561,4 +561,4 @@
         ;;
 esac
 
-exit 0
\ No newline at end of file
+exit 0
diff -urN sonarqube-5.3/conf/wrapper.conf sonarqube-5.3-mod/conf/wrapper.conf
--- sonarqube-5.3/conf/wrapper.conf	2016-02-23 23:05:37.000000000 +0800
+++ sonarqube-5.3-mod/conf/wrapper.conf	2016-02-23 23:10:53.000000000 +0800
@@ -8,16 +8,21 @@
 # DO NOT EDIT THE FOLLOWING SECTIONS
 #
 
-
+set.default.SONAR_HOME=/usr/share/sonar
 #********************************************************************
 # Wrapper Java
 #********************************************************************
 wrapper.java.additional.1=-Djava.awt.headless=true
 wrapper.java.mainclass=org.tanukisoftware.wrapper.WrapperSimpleApp
-wrapper.java.classpath.1=../../lib/jsw/*.jar
-wrapper.java.classpath.2=../../lib/*.jar
-wrapper.java.library.path.1=./lib
+wrapper.java.classpath.1=%SONAR_HOME%/lib/jsw/*.jar
+wrapper.java.classpath.2=%SONAR_HOME%/lib/*.jar
+wrapper.java.library.path.1=%SONAR_HOME%/lib
 wrapper.app.parameter.1=org.sonar.application.App
+wrapper.app.parameter.2=-Dsonar.path.home=%SONAR_HOME%
+wrapper.app.parameter.3=-Dsonar.path.logs=/var/log/sonar
+wrapper.app.parameter.4=-Dsonar.path.data=/var/lib/sonar/data
+wrapper.app.parameter.5=-Dsonar.path.log=/var/log/sonar
+wrapper.app.parameter.6=-Dsonar.path.temp=/var/lib/sonar/temp
 wrapper.java.initmemory=3
 wrapper.java.maxmemory=3
 
@@ -27,7 +32,7 @@
 
 wrapper.console.format=PM
 wrapper.console.loglevel=INFO
-wrapper.logfile=../../logs/sonar.log
+wrapper.logfile=/var/log/sonar/sonar.log
 wrapper.logfile.format=M
 wrapper.logfile.loglevel=INFO
 
@@ -85,4 +90,4 @@
 wrapper.disable_restarts=TRUE
 wrapper.ping.timeout=0
 wrapper.shutdown.timeout=3000
-wrapper.jvm_exit.timeout=3000
\ No newline at end of file
+wrapper.jvm_exit.timeout=3000
